docs/releases/overview.md

```diff
@@ -26,8 +26,31 @@ See the full release notes on [GitHub ↗](https://github.com/vegaprotocol/vega/
 ## Vega core software
 The Vega core software is public on a business-source licence, so you can both view the repository change logs, and refer here for summary release notes for each version that the validators use to run the Vega mainnet. Releases are listed with their semantic version number and the date the release was made available to mainnet validators.
 
+### Pre-release Versions 0.65.0, 0.65.1 and 0.65.2 combined | 2022-12-23
+This version was released to the Vega testnet on 23 December 2022.
 
-### Pre-release Versions 0.63.0, 0.63.1, 0.63.2 and 0.64.0 combined | 2022-11-11
+The final scheduled testnet release of 2022 has introduced some refactors to the liquidity provision (LP) code. The protocol doesn’t want LP margins to be affected by the probability of trading, because this makes the job of being an LP overly difficult. Similarly, the protocol wants to discourage LPs from posting orders too far away from the mid price. This ensures that liquidity provided is 'useful' liquidity. Doing this refactor now addresses these points before liquidity providers start to integrate with the protocol, both for the mainnet sims and the Alphap Mainnet release.
+
+Improvements have been made to the way snapshot data is stored and retrieved, this is in order to manage the memory load during these processes. This combined with some Postgres configuration improvements has made significant improvements to the network stability. This will be further proved out in the incentive running over the coming week.
```

Comment:
```suggestion
The way snapshot data is stored and retrieved has been improved to manage the memory load during these processes. This combined with some Postgres configuration changes has made significant improvements to the network stability. This will be further proved out in the incentive running over the coming weeks.
```